spec/unit/deployment/config/loader_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+require 'r10k/deployment/config/loader'
+
+describe R10K::Deployment::Config::Loader do
+
+  it "creates R10K::Deployment::Config::Loader instances" do
```

Comment:
No problem removing that. It was the first "are these tests working" test. I didn't see a problem with leaving it in, but happy to remove it.